src/ArToolkitContext.ts

```diff
@@ -1,17 +1,17 @@
 import * as THREE from 'three'
-import { IContextParameters } from './CommonInterfaces/THREEx-interfaces';
+import { IArToolkitContext, IContextParameters } from './CommonInterfaces/THREEx-interfaces';
 import { ArMarkerControls } from "./ArMarkerControls";
 import jsartoolkit from "@ar-js-org/artoolkit5-js"; // TODO comment explanation
 const { ARController } = jsartoolkit;
 
 //namespace THREEx {
-export class ArToolkitContext {
+export class ArToolkitContext implements IArToolkitContext{
     private _updatedAt: any;
     public parameters: IContextParameters;
     public arController: typeof ARController;
     private initialized: boolean;
     private _arMarkersControls: any;
-    private _artoolkitProjectionAxisTransformMatrix: any;
+    public _artoolkitProjectionAxisTransformMatrix: any;
```

Comment:
the name suggest that is a private method, but it's not so redeclared as public.